utopia-remix/.env.sample

```diff
@@ -1,26 +1,23 @@
-APP_ENV="local"
-BACKEND_URL="http://127.0.0.1:8002"
-CORS_ORIGIN="http://localhost:8000"
-DATABASE_URL="postgres://<username>:postgres@localhost:5432/utopia"
-REACT_APP_EDITOR_URL="http://localhost:8000"
+# APP_ENV="local"
+# BACKEND_URL="http://127.0.0.1:8002"
+# CORS_ORIGIN="http://localhost:8000"
+# REACT_APP_EDITOR_URL="http://localhost:8000"
```

Comment:
I believe these two (plus `UTOPIA_CDN_URL`) will always be the same, so unless I'm missing something there's no need for multiple variables

Reply:
[`d8f6218` (#5034)](https://github.com/concrete-utopia/utopia/pull/5034/commits/d8f621809da9058ff7b5c0aa90a620216c7e0493)

Reply:
the CDN URL can be different on prod tho